plugins/node/opentelemetry-instrumentation-redis/src/utils.ts

```diff
@@ -49,17 +28,14 @@ const endSpan = (span: Span, err?: Error | null) => {
   span.end();
 };
 
-export const getTracedCreateClient = (tracer: Tracer, original: Function) => {
+export const getTracedCreateClient = (original: Function) => {
```

Comment:
reviewer note: removing unused parameter (which is also locked to it's initial value and does not update on `setTracerProvider`